apps/openchallenges/db-cli/src/config.ts

```diff
@@ -6,4 +6,10 @@ export const config = {
     },
     uri: 'mongodb://localhost:27017/openchallenges',
   },
+  mariadb: {
+    host: 'localhost',
```

Comment:
@tschaffter FYI, I still couldn't connect to the db any other way than `localhost`. Do you have any additional ideas?

Reply:
Did you try to substitute `localhost` by `openchallenges-mariadb`? This domain name should be defined in your `/etc/hosts` as defined [here](https://github.com/Sage-Bionetworks/sage-monorepo/blob/c91e6163bded6370eaf9c437e83370a6dcad1d86/tools/configure-hostnames.sh#L14).

Reply:
Yes, I did.

Reply:
lakdjflkja apologies, I never actually tested `openchallenges-mariadb` using the CLI -- I've been testing the credentials directly with my test TS script.

Using `openchallenges-mariadb` as the host works!

![Screen Shot 2023-02-14 at 10 43 29 AM](https://user-images.githubusercontent.com/9377970/218828694-6e48be8d-c658-47d7-a012-88435a554f94.png)